core/src/main/resources/reference.conf

```diff
@@ -409,7 +426,7 @@ database {
   #driver = "slick.driver.MySQLDriver$"
   #db {
   #  driver = "com.mysql.jdbc.Driver"
-  #  url = "jdbc:mysql://host/cromwell"
+  #  url = "jdbc:mysql://host/cromwell?rewriteBatchedStatements=true"
```

Comment:
Disagree on inconsistency. This setting is simply saying if anything is intended to be a batch insert to allow it. The other setting is how we batch one particular setting.

More importantly however I wanted to make it as hard as possible for someone to not accidentally leave this off (it doesn't hurt anything AFAIK to leave it on), considering how many people just copy/paste this stuff that's likely.